portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase

```diff
@@ -3952,39 +4088,205 @@ definition {
 	}
 
 	@description = "LPS-86285: Validate in the content page if, the user can discard the section when he is editing the experiences (not published)	"
-	@ignore = "true"
+	@ignore = "false"
 	@priority = "3"
-	test DiscardingASectionOfAnUnpublishedExperiment {
+	test DiscardingASectionOfAnUnpublishedExperience {
+		task ("Go to Experience Content Page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Experience Content Page",
+				siteName = "Test Site Name");
+		}
 
-		// TODO LPS-86285 DiscardingASectionOfAnUnpublishedExperiment pending implementation
+		task ("Create an experience, add a component and make changes without publish it") {
+			PageEditor.addFragment(
+				collectionName = "Layout Elements",
+				fragmentName = "Container");
+
+			Button.clickPublish();
+
+			PageEditor.addExperience(
+				experienceName = "New Experience",
+				segmentName = "Anyone");
+		}
+
+		task ("Go to the pages menu and discard changes") {
+			ProductMenu.gotoPortlet(
+				category = "Site Builder",
+				portlet = "Pages");
+
+			PageEditor.clickDiscardDraft(pageName = "Experience Content Page");
+		}
 
+		task ("Go to Experience Content Page and check the Experience is not present") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Experience Content Page",
+				siteName = "Test Site Name");
+
+			PageEditor.viewFragment(fragmentName = "Container");
+
+			Click(locator1 = "PageEditor#EXPERIENCE_LABEL");
+
+			AssertElementNotPresent(
+				key_experienceName = "New Experience",
+				locator1 = "PageEditor#EXPERIENCE_SELECT_ITEM");
+		}
 	}
 
 	@description = "LPS-86285: Validate in the content page if, the user can discard the changes when he is editing the experiences (not published)"
-	@ignore = "true"
+	@ignore = "false"
 	@priority = "3"
 	test DiscardingChangesFromUnpublishedExperiences {
+		task ("Go to Experience Content Page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Experience Content Page",
+				siteName = "Test Site Name");
+		}
+
+		task ("Create a component and add a component without publishing them") {
+			PageEditor.addFragment(
+				collectionName = "Basic Components",
+				fragmentName = "Heading");
+
+			Button.clickPublish();
+
+			PageEditor.addExperience(
+				experienceName = "New Experience",
+				segmentName = "Anyone");
 
-		// TODO LPS-86285 DiscardingChangesFromUnpublishedExperiences pending implementation
+			Click(locator1 = "PageEditor#EXPERIENCE_LABEL");
```

Comment:
No need

Reply:
If I remove it the test fails with this error: 
BUILD FAILED: Element is not present at "//div[contains(@class,'cke_editable_inline') and contains(@id,'_content')] | //div[contains(@class,'form-group')]//textarea[contains(@id,'_content')]"
     [exec]     /Users/robertstefan/Desktop/Liferay/liferay-portal/portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro[editFragmentText]:2171
     [exec]     /Users/robertstefan/Desktop/Liferay/liferay-portal/portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase[DiscardingChangesFromUnpublishedExperiences]:4159
I think is because when an Experience is added without clicking on Publish, the Selector of Experiences doesn't close so the test can't continue with the next function.

Reply:
You're right